app/scripts/background.ts

```diff
@@ -12,7 +12,13 @@ const onClickContextMenus = async (
   }
   const imageUrl = info.srcUrl
   const teamName = prompt(chrome.i18n.getMessage('promptTeamName')) || ''
```

Comment:
This code can do `undefined` check by `if` statement , So you can skip the process of assigning `''` with the ternary operator.
Please remove ` || ''`.

Reply:
sore ha sou